.github/workflows/auto_airac_actions.yml

```
@@ -0,0 +1,48 @@
+name: Automatic AIRAC Actions
+
+on:
+  push:
+    branches:
+      - main
+  repository_dispatch:
+    types: [auto_airac_actions]
+
+jobs:
+  python-script-and-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: pip install -r .github/workflows/requirements.txt
+
+      - name: Run Python script
+        env:
+            REMOTE_KEY: ${{ secrets.DOWNLOAD_ARTIFACT }}
```

```suggestion
            REMOTE_KEY: ${{ secrets.GITHUB_TOKEN }}
```
Using the `GITHUB_TOKEN` should authenticate the API request that this is being used for without the need for a specific PAT.

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow